src/main/java/net/neoforged/moddevgradle/dsl/RunModel.java

```diff
@@ -77,6 +79,12 @@ public String getName() {
      */
     public abstract Property<String> getIdeName();
 
+    /**
+     * Whether an IDE run config should be generated, alternatively
+     * set the IDE name to {@code ""}.
+     */
+    public abstract Property<Boolean> getIdeConfigGenerated();
```

Comment:
Instead of the property, to reduce the amount of special cases, we could add a method to mark a run as not having an ide config which in the backend just sets the idename to empty